webdriver/src/com/blazemeter/jmeter/webdriver/phantomjs/PhantomJSDriverConfig.java

```diff
@@ -34,6 +36,13 @@ protected Capabilities createCapabilities() {
         capabilities.setCapability(CapabilityType.PROXY, createProxy());
         capabilities.setCapability(PhantomJSDriverService.PHANTOMJS_EXECUTABLE_PATH_PROPERTY,
                 getPhantomJsExecutablePath());
+        String cliArgs = JMeterContextService.getContext().getVariables().get("phantomjs.cli.args");

```

Comment:
Why variables from the context? This does not look right. CLI args meant to be taken at the beginning of the test, variables are overkill.